src/Integration/ProfileConflicts/ConflictsManager.cs

```diff
@@ -57,11 +62,23 @@ public ConflictsManager(IServiceProvider serviceProvider, ILogger logger)
 
         public IReadOnlyList<ProjectRuleSetConflict> GetCurrentConflicts()
         {
+            var configProvider = this.serviceProvider.GetService<IConfigurationProvider>();
```

Comment:
This class is only used by the _RuleSetConflictsController_ so I could have put the check there.
However, it was easier to put it here as it required less refactoring, and there was already a defensive _bound project is not null_ check.